lib/systems/inspect.nix

```diff
@@ -13,8 +14,8 @@ rec {
     BigEndian    = { cpu = { significantByte = significantBytes.bigEndian; }; };
     LittleEndian = { cpu = { significantByte = significantBytes.littleEndian; }; };
 
-    Unix         = { kernel = { families = { inherit (kernelFamilies) unix; }; }; };
     BSD          = { kernel = { families = { inherit (kernelFamilies) bsd; }; }; };
+    Unix         = [ BSD Linux SunOS Hurd Cygwin ];
```

Comment:
I removed the redundant `patterns.`, aside from the string indexed ones.  I'm not aware of a better way to deal with them.  I don't really have strong feelings on this vs. `with patterns;`, but this is slightly more concise.